.github/actions/spelling/patterns.txt

```diff
@@ -141,3 +141,15 @@ regexp?\.MustCompile\(`[^`]*`\)
 
 # Don't check names in dependabot.yaml reviewers section
 ^\s+reviewers:\s*\[\s*"(?:Nino-K|ericpromislow)"\s*\]
+
+# Directives to avoid extension-related publisher names and repositories:
+
+# Directives to skip a full line
+^.*spellcheck-ignore-line.*$
+
+# Ignore extension names (usually based on a github repository)
+^\s*\bslug:\s+'.*?/.*',\s*$
+^\s*\bpublisher:\s+\{\s*name:\s*'.?*'.*?\}
+
+# Ignore extension repository account names (no slash in the quoted part)
```

Comment:
Do we want to enforce the lack of a slash (`[^/]*` instead of `.*`)?

Reply:
I redid the comments to clarify what these lines do.